api/liquidate.ts

```diff
@@ -3,11 +3,24 @@ import colors from "colors";
 import { Vault__factory, PositionManager__factory } from "@mycelium-ethereum/perpetual-swaps-contracts";
 import getOpenPositions from "../src/helpers/getOpenPositions";
 import getPositionsToLiquidate from "../src/helpers/getPositionsToLiquidate";
-import { sleep } from "../src/helpers/helpers";
 
 const liquidationHandler = async function () {
     try {
-        const provider = new ethers.providers.JsonRpcProvider(process.env.RPC_URL);
+        const primaryProvider = new ethers.providers.JsonRpcProvider(process.env.RPC_URL);
+        const fallbackProvider = new ethers.providers.JsonRpcProvider(process.env.FALLBACK_RPC_URL);
+        const provider = new ethers.providers.FallbackProvider([
+            {
+                priority: 1,
+                provider: primaryProvider,
+                stallTimeout: 1500,
+            },
+            {
+                priority: 2,
+                provider: fallbackProvider,
+                stallTimeout: 1500,
+            },
+        ]);
```

Comment:
If `primaryProvider` starts messing up, will this switch to `fallbackProvider`? I'm reading the docs and it seems to suggest that a quorum is required before making a decision so I'm wondering if this will just start failing if one of them does.

I was originally suggesting something a bit less elegant where we switch between 2-3 on errors/bad responses (based on [this python snippet](https://pastebin.com/Ftuhk6m1)), but if this works too them this is clearly much better.